src/tests/ftest/launch.py

```diff
@@ -897,6 +901,23 @@ def archive_logs(avocado_logs_dir, test_yaml, args):
     spawn_commands(host_list, "; ".join(commands), 900)
 
 
+def get_log_size(test_yaml, args):
+    """Get the size of the the directoy of the host test log files in
+    the avocado results directory and store the values in a file.
+
+    Args:
+        test_yaml (str): yaml file containing host names
+        args (argparse.Namespace): command line arguments for this program
+    """
+    logs_dir = os.environ.get("DAOS_TEST_LOG_DIR", DEFAULT_DAOS_TEST_LOG_DIR)
+    log_size_file = os.path.join(logs_dir, "log_size.log")
+    host_list = get_hosts_from_yaml(test_yaml, args)
+
+    # Create a file that contains the sizes on each test log and the logs_dir.
+    command = "du -h {} &> {}".format(logs_dir, log_size_file)
```

Comment:
It might be interesting to know the sizes of individual logs, not just the size of the dir they consume all added together.

Reply:
The command lists out the size of the files inside the directory and also the size of the directory itself.

Reply:
I must be missing something.  I don't see any file listing command, just the `du -h` above.

Reply:
No you are right, I missed it when I tested the command. Fixed.